src/sinks/util/retries.rs

```diff
@@ -65,8 +66,11 @@ where
                     return None;
                 }
 
-                if self.logic.should_retry_response(response) {
-                    warn!(message = "retrying after response.");
+                if let Some(ref reason) = self.logic.should_retry_response(response) {
+                    warn!(
+                        message = "retrying after response.",
+                        reason = tracing::field::display(reason),
```

Comment:
```suggestion
                        %reason,
```

Reply:
This probably needs to be fmt as well

Reply:
I'm not clear what you mean by "needs to be fmt"

Reply:
Sorry, I mean't that I think if you applied this change via the github UI the rustfmt check would fail since I think it would want to collapse this into one line.

Reply:
Got it, done.